include/kernel.h

```diff
@@ -654,7 +654,7 @@ struct _static_thread_data {
  * @param p3 3rd entry point parameter.
  * @param prio Thread priority.
  * @param options Thread options.
- * @param delay Scheduling delay (in milliseconds), zero for no delay and K_FOREVER for delayed start.
+ * @param delay Scheduling delay (in milliseconds), zero for no delay, K_FOREVER for delayed start.
```

Comment:
wouldn't jus be better break the line ? It seems gramatically incorrect  but I may be wrong :p